tests/foreman/longrun/test_leapp_upgrade.py

```diff
@@ -0,0 +1,55 @@
+"""Test for Upgrade using leapp
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: LEAPP
+
+:Assignee: shwsingh
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+
+
+def pre_upgrade_check(target_sat, target_version):
+    target_sat.execute(f'leapp preupgrade --target {target_version}')
+    target_sat.execute('yum install cockpit-leapp')
+
+
+def post_upgrade_cleanup_rhel7(target_sat, version):
+    # remove all packages from the exclude list
+    target_sat.execute('dnf config-manager --save --setopt exclude=' '')
```

Comment:
is this just some weird formatting?
```suggestion
    target_sat.execute('dnf config-manager --save --setopt exclude=')
```